src/routes/index.ts

```diff
@@ -269,4 +270,9 @@ router.route(config.UPDATE_CHECK_YOUR_ANSWERS_URL)
   .get(updateCheckYourAnswers.get)
   .post(updateCheckYourAnswers.post);
 
+router.route(config.UPDATE_DUE_DILIGENCE_URL)
+  .all(authentication)
+  .get(updateDueDiligence.get)
+  .post(...validator.dueDiligence, checkValidations, updateDueDiligence.post);
```

Comment:
Need navigation.hasUpdatePresenter to check up on how we get to it

Reply:
see navigation.hasPresenter

Reply:
@AndreKramer7654 added a new nav check, hasWhoIsMakingUpdate.